### PR TITLE
🏗 Sync versioning.json to CDN

### PR DESCRIPTION
**Pull request by danielrozenberg**
Followup on #36714
Part of #36152

### CODE REVIEW

File: .github/workflows/sync-versioning-to-cdn.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Install Dependencies
+        run: npm i -g @cloudflare/wrangler
+
+      - name: ⭐ Sync versioning.json to the CDN ⭐
+        run: |
```
Comment (rsimha) [inline .github/workflows/sync-versioning-to-cdn.yml:27]: For maintainability / readability, can we move this business logic into a shell script and invoke it from this file?

**Review by rsimha — APPROVED** [review]

File: .github/workflows/sync-versioning-to-cdn.yml (hunk 1 above)
Comment (danielrozenberg) [reply .github/workflows/sync-versioning-to-cdn.yml:27]: None of the other files in this directory use this pattern - I agree it's a worthwhile endeavour, maybe we can switch all of them in a followup PR?

Comment (rsimha) [reply .github/workflows/sync-versioning-to-cdn.yml:27]: Absolutely, SGTM. 👍

**Comment by danielrozenberg** [comment]
Converting to draft until #36714 is submitted

**Pull request merged**